src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderDatesResource.java

```diff
@@ -74,6 +73,23 @@ public ResponseEntity<OffenderCalculatedKeyDates> getOffenderKeyDates(@PathVaria
             .body(offenderDatesService.getOffenderKeyDates(bookingId));
     }
 
+
+    @ApiResponses({
+        @ApiResponse(responseCode = "200", description = "Offender key dates found", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = OffenderCalculatedKeyDates.class))}),
+        @ApiResponse(responseCode = "400", description = "Invalid request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "403", description = "Forbidden - user not authorised to get an offender's calculated dates", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})
+    })
+    @Operation(summary = "Get the key dates for an offender.", description = "Requires RELEASE_DATES_CALCULATOR")
+    @GetMapping("/sentence-calculation/{offenderSentCalcId}")
+    @PreAuthorize("hasRole('RELEASE_DATES_CALCULATOR')")
+    @ProxyUser
+    public ResponseEntity<OffenderCalculatedKeyDates> getOffenderKeyDatesForOffenderSentCalcId(@PathVariable("offenderSentCalcId") @Parameter(description = "The Offender Sent Calc Id id of offender", required = true) final Long offenderSentCalcId) {
+        return ResponseEntity.status(HttpStatus.OK)
+            .body(offenderDatesService.getOffenderKeyDatesByOffenderSentCalcId(offenderSentCalcId));
+    }
```

Comment:
very minor but you can just return OffenderCalculatedKeyDates - no need for the ResponseEntity wrapper.